muffnn/__init__.py

```diff
@@ -3,8 +3,10 @@
 from .autoencoder.autoencoder import Autoencoder
 from .fm.fm_classifier import FMClassifier
 from .core import TFPicklingBase
+from .version import __version__
 
-__all__ = ['MLPClassifier',
+__all__ = ['__version__',
+           'MLPClassifier',
```

Comment:
I think it might be good leave it out. I don’t think people expect the version when they import *.

Reply:
ah, good catch. i was misunderstanding how `__all__` works.